addons/website_slides/models/slide_channel.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+<<# -*- coding: utf-8 -*-
```

Comment:
In commit about "upload a channel" there are some leftover "<<" probably removed in another commit.